packages/react/src/dialog-title/DialogTitle.tsx

```diff
@@ -0,0 +1,36 @@
+import * as RadixDialog from "@radix-ui/react-dialog";
+import { type ComponentPropsWithRef, forwardRef } from "react";
+
+import type { ExtendProps } from "../utils";
+
+import { Flex } from "../flex";
+import { extractSprinkles } from "../sprinkles";
+
+type FooterProps = ExtendProps<
+  ComponentPropsWithRef<typeof Flex>,
+  ComponentPropsWithRef<typeof RadixDialog.Title>
+>;
+
+export const DialogTitle = forwardRef<HTMLDivElement, FooterProps>(
+  ({ children, ...props }, ref) => {
+    const { restProps, sprinkleProps } = extractSprinkles(props);
+    return (
+      <Flex
```

Comment:
let's use `<Heading />` with the correct level prop here instead of manually setting the font size